docs/components/badges.html

```diff
@@ -0,0 +1,164 @@
+                        {% endif %}
+                    </td>
+                    <td>{{ item.description }}</td>
+                </tr>
+            {% endfor %}
+        </tbody>
+    </table>
+</section>
+
+<section class="stacks-section">
+    <h2 class="grid jc-space-between ai-center stacks-h2"><div class="grid--cell fl1">Examples</div></h2>
+    <h3 class="grid jc-space-between ai-center stacks-h3"><div class="grid--cell fl1">Badges (Default)</div></h3>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<span class="s-badge">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeGold.svg" aria-hidden="true"></span>
+    <span>Great Question</span>
+</span>
+<span class="s-badge">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeSilver.svg" aria-hidden="true"></span>
+    <span>Favorite Question</span>
+</span>
+<span class="s-badge">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeBronze.svg" aria-hidden="true"></span>
+    <span>Altruist</span>
+</span>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid gs4">
+                <span class="grid--cell s-badge">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeGold.svg" aria-hidden="true"></span>
+                    <span>Great Question</span>
+                </span>
+                <span class="grid--cell s-badge">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeSilver.svg" aria-hidden="true"></span>
+                    <span>Favorite Question</span>
+                </span>
+                <span class="grid--cell s-badge">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeBronze.svg" aria-hidden="true"></span>
+                    <span>Altruist</span>
+                </span>
+            </div>
+        </div>
+    </div>
+
+    <h3 class="grid jc-space-between ai-center stacks-h3"><div class="grid--cell fl1">Tag Badge</div></h3>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<span class="s-badge s-badge__tag">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeGold.svg" aria-hidden="true"></span>
+    <span>css</span>
+</span>
+<span class="s-badge s-badge__tag">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeSilver.svg" aria-hidden="true"></span>
+    <span>javascript</span>
+</span>
+<span class="s-badge s-badge__tag">
+    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeBronze.svg" aria-hidden="true"></span>
+    <span>swift</span>
+</span>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid gs4">
+                <span class="grid--cell s-badge s-badge__tag">
+                    <span class="s-badge--image"><img src="../{{ site.img }}/BadgeGold.svg" aria-hidden="true"></span>
+                    <span>css</span>
```

Comment:
I don't think we need the `<span>`s surrounding the text. We should included as little markup as possible in our examples.